mslice/tests/slice_test.py

```diff
@@ -0,0 +1,110 @@
+from mock import MagicMock, patch
+import unittest
+
+from mslice.models.slice.slice import Slice
+from mantid.simpleapi import AnalysisDataService
+
+class SliceTest(unittest.TestCase):
+
+    @classmethod
+    def tearDown(cls) -> None:
+        AnalysisDataService.clear()
```

Comment:
```suggestion
    def tearDown(self) -> None:
        AnalysisDataService.clear()
```

Reply:
Resolved as this change is anticipated to have no impact.